cabal-testsuite/src/Test/Cabal/Monad.hs

```diff
@@ -51,14 +51,19 @@ module Test.Cabal.Monad (
     skipUnless,
     skipUnlessIO,
     -- * Known broken tests
-    expectedBroken,
-    unexpectedSuccess,
+    expectBroken,
+    expectBrokenIf,
+    expectBrokenUnless,
+    -- * Flaky tests
+    flaky,
+    flakyIf,
```

Comment:
You will need to update the documentation in https://github.com/haskell/cabal/tree/master/cabal-testsuite#frequently-asked-questions

Reply:
Done